libs/utils/decorate.js

```diff
@@ -117,6 +117,29 @@ export const decorateBlockHrs = (el) => {
   if (hasHr && els.length) el.classList.add('has-divider');
 };
 
+export const decorateBgContent = (el) => {
+  const els = el.querySelectorAll('p');
+  let insidePattern = false;
+  let decoratedBlock;
+  [...els].forEach((e) => {
```

Comment:
For evergreen browsers, there's no need to convert `NodeList` into an array in order to use `forEach` method: https://developer.mozilla.org/en-US/docs/Web/API/NodeList/forEach